assesspy/ci.py

```diff
@@ -1,38 +1,48 @@
-# Import necessary libraries
+import math
+from typing import Union
+
 import pandas as pd
-from pandas.api.types import is_numeric_dtype
+import statsmodels.api as sm
 
-from .formulas import cod, prd
+from .metrics import cod, prd
 from .utils import check_inputs
 
 
-def boot_ci(fun, nboot=100, alpha=0.05, **kwargs):
+def boot_ci(
+    fun,
+    estimate: Union[list[int], list[float], pd.Series],
+    sale_price: Union[list[int], list[float], pd.Series],
```

Comment:
Now that all the metrics share the same two input args, we no longer need to mess with the complication of `kwargs`.